packages/webpack/src/config/base.js

```diff
@@ -96,6 +97,8 @@ export default class WebpackBaseConfig {
 
   getBabelOptions () {
     const envName = this.name
+    const { corejs = 'auto' } = this.buildContext.buildOptions
```

Comment:
We should add defaults to `@nuxt/config/src/config/build.json` (also update types with `yarn jest packages/config -u`)